subsys/bluetooth/host/hci_core.c

```diff
@@ -1737,6 +1742,10 @@ int bt_unpair(uint8_t id, const bt_addr_le_t *addr)
 		return 0;
 	}
 
+	if (!addr) {
+		addr = BT_ADDR_LE_ANY;
+	}
```

Comment:
What does this do? 

From I can tell, `unpair` has no handling for for `BT_ADDR_LE_ANY` so I am not convinced this change makes any sense in the current state.

Reply:
Changed to not run unpair with BT_ADDR_LE_ANY or NULL